timbermill-java/timbermill-api/src/main/java/com/datorama/oss/timbermill/unit/Event.java

```diff
@@ -288,7 +288,7 @@ private int getNumberMapSize(Map<String, Number> map) {
 					size += entry.getKey().length();
 				}
 				if (entry.getValue() != null) {
-					size += (Math.log10(entry.getValue().doubleValue()) + 1) + 4; // "":,
+					size += (entry.getValue().toString().length() + 1) + 4; // "":,
```

Comment:
@edengal How big this value can be? entry.getValue().toString()
Please surround this code with exception handling.

Reply:
@janevelger How long the string will be? 1.7976931348623157E308 (Max double value)

Reply:
@janevelger Can you elaborate on which exception handling you are referring to?